packages/contracts/src/dollar/ERC20Ubiquity.sol

```diff
@@ -34,15 +29,15 @@ contract ERC20Ubiquity is
     // ----------- Modifiers -----------
     modifier onlyMinter() {
         require(
-            manager.hasRole(manager.GOVERNANCE_TOKEN_MINTER_ROLE(), msg.sender),
+            manager.hasRole(manager.UBQ_MINTER_ROLE(), msg.sender),
```

Comment:
Same as above.

Reply:
Changed them all to GOVERNANCE_TOKEN_MINTER_ROLE